translator/source_code/icone_translator.py

```diff
@@ -65,9 +80,13 @@ def wzdx_creator(messages, info):
         # print("")
         # print("")
         # Parse Incident to WZDx Feature
-        wzd['features'].append(parse_incident(incident))
-
+        polyline = parse_polyline(incident['location']['polyline'])
+        avg_point = [(polyline[0][0] + polyline[-1][0]) / 2, (polyline[0][1] + polyline[-1][1]) / 2]
 
+        if is_in_colorado(avg_point) or True:
```

Comment:
What are we doing with this check? It looks like we're checking if the average point on the polyline is in Colorado, then proceeding regardless. If that's the case, do we even need this check? Or the `avg_point` calculation for that matter?

Reply:
We will remove this and it was there before we knew that iCone was pre-filtering the data for Colorado.

Reply:
What Abinash said. We added this as an attempt to filter the data for just Colorado. The incidents_extended data we have been working with is not filtered by state, and so it made sense at the time. 
After implementing this geo-filter, we realized that none of the data was from Colorado. We wanted some data to test with, so we decided to un-do the geo-filter. 
Instead of commenting it out or deleting it, we decided to leave it active and unit test it, in case we wanted to re-implement it later. 

Now that we know iCone is geo-filtering the data for us we can remove it

Reply:
Doesn't the `or True` portion here allow all data through regardless if its in Colorado or not? Moot since it will be removed I suppose though.

Reply:
Right, we just wanted to disable it in a way that would be easy to re-enable later. Not best practice for sure but it made sense at the time

Reply:
Ok, I'm up to speed with you now. Thanks!

Reply:
The code has been removed.